packages/services/src/staking-service/get-validator-info.ts

```diff
@@ -0,0 +1,18 @@
+import { Impl } from '.';
+import { servicesCtx } from '../ctx/prax';
+import { Code, ConnectError } from '@connectrpc/connect';
+
+export const getValidatorInfo: Impl['getValidatorInfo'] = async (req, ctx) => {
```

Comment:
I added the `getValidatorInfo` implementation because it would not be optimal to iterate through the list of all validators for each unbonding balance

Reply:
This is great 👍